frontend/packages/knative-plugin/src/components/overview/KnativeOverview.tsx

```diff
@@ -2,26 +2,35 @@ import * as React from 'react';
 import { ResourceSummary } from '@console/internal/components/utils';
 import { OverviewItem, PodRing } from '@console/shared';
 import { RevisionModel } from '../../models';
+import { usePodsForRevisions } from '../../utils/usePodsForRevisions';
 
 type KnativeOverviewProps = {
   item?: OverviewItem;
 };
 
+export const KnativeOverviewRevisionPodsRing: React.FC<KnativeOverviewProps> = ({ item }) => {
+  const { obj } = item;
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  const revisions = React.useMemo(() => [obj], [obj.metadata.uid]);
```

Comment:
Whenever you disable any lint rules, please add a comment as to why.
This is a bit odd. If `usePodsForRevisions` only needs the uid's of the revisions, the API should probably be changed.